src/python/pants/core/goals/lint.py

```diff
@@ -84,7 +85,11 @@ def metadata(self) -> dict[str, Any]:
         return {"partition": self.partition_description}
 
     def level(self) -> LogLevel | None:
-        return LogLevel.ERROR if self.exit_code != 0 else LogLevel.INFO
+        if self._render_message:
```

Comment:
do you mean `if not self._render_message`? In that case, it should also probably be TRACE

Reply:
I'm tempted to leave it at `debug` to help us debug issues in the converter. If anything, happy to hear a better name. Thoughts?